test/Utility/TextProcessingTraitTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\ProjectAanvraag\Utility;
+
+use PHPUnit\Framework\TestCase;
+
+class TextProcessingTraitTest extends TestCase
+{
+    use TextProcessingTrait;
+
+    /**
+     * @test
+     */
+    public function it_does_not_trim_closing_html_tags(): void
+    {
+        // Use an already transformed description to simulate the creation of a summary from a description.
+        $description = 'À la périphérie bruxelloise, le parc déploie ses pelouses, ses bois, ses arbres remarquables, ses massifs de rhododendrons,... plus de 450 espèces de plantes sauvages sont recensées sur le site.<br />
```

Comment:
Looks good to me!
In the ticket the problem occurs with `\n`. Did you also test that or is the json being transformed the html in an earlier stage?

Reply:
The `\n` from the original description is first transformed to `<br />`. So the method to generate a HTML summary description works on `<br />` instead.

Reply:
This is the line that does the replacement first https://github.com/cultuurnet/projectaanvraag-silex/blob/master/src/Widget/Twig/TwigPreprocessor.php#L215